backend/plugins/posclient_api/src/modules/posclient/graphql/resolvers/customResolvers/order.ts

```
@@ -0,0 +1,179 @@
+import { IContext } from '@/posclient/@types/types';
+import { IOrderItemDocument } from '~/modules/posclient/@types/OrderItems';
+import { IOrderDocument } from '~/modules/posclient/@types/orders';
+import { IEbarimtDocument } from '~/modules/posclient/@types/putResponses';
+// import { sendSalesMessage, sendCoreMessage } from '../../messageBroker';
+import { fakePutData } from '@/posclient/utils/orderUtils';
+import { sendTRPCMessage } from 'erxes-api-shared/utils';
+
+export default {
+  async items(order: IOrderDocument, {}, { models }: IContext) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)